src/emulate.c

```diff
@@ -615,7 +504,27 @@ void rv_step(riscv_t *rv, int32_t cycles)
         }
         last_pc = rv->PC;
 
-        /* execute the block */
+#if RV32_HAS(JIT)
+        /* execute the block by JIT compiler */
+        uint8_t *code = NULL;
+        if (block->hot)
+            code = code_cache_lookup(rv->cache, block->pc_start);
+        if (!code) {
+            /* check if using frequency of block exceed threshold */
+            if ((block->hot = cache_hot(rv->cache, block->pc_start)))
+                code = block_compile(rv);
+        }
+        if (block->hot) {
+            /* execute machine code */
+            /* clang-format off */
```

Comment:
Why did you add clang-format tweak here?

Reply:
After using `clang-format-12`, this line would become `(exec_block_func_t) code) (rv, block->ir)`. However, it fails the coding style check because the expected format is without a space `(exec_block_func_t) code)(rv, block->ir)`.

Reply:
You can simply rewrite as following:
```c
exec_block_func_t _code = (exec_block_func_t) code;
if (unlikely(_code(rv, block->ir)))
```